### DISCUSSION TITLE
Any plans to support Arabic (ar-*) Language?

### DISCUSSION THREAD

**Discussion by aqasemi** [deepgram/community#298] [Feature Requests]
Hello there.

Are you planning on supporting more languages, Arabic in particular, for Nova model?

**Comment by jjmaldonis** [comment]
Hey @alqasemy2020, we are planning to support additional languages for Nova, and Arabic is on the list. However, we do not have an estimate for when it will be available.

**Comment by Bohdananas** [comment]
Hmm...

**Comment by arbaaz** [comment]
Any updates on this? or Any hack?

**Comment by mohammad-melhem** [comment]
Any updates on this?

**Comment by arjunchakraborti** [comment]
I saw that Deepgram has joined hands with Intella to deliver Arabic language for STT - isnt it live yet?
